src/Tribe/Views/V2/View.php

```diff
@@ -1274,6 +1274,9 @@ protected function setup_template_vars() {
 
 		$events = (array) $this->repository->all();
 
+		// are we paginating
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 5a36f39403e69e6db147ad869b14cde8</sub>